cutter build fails with `-Werror=odr`

**Environment information**
* Operating System: Gentoo
* Cutter version: 
```
commit bffe193708ecb1ad3c24a31893f5cb26a8b0ee13 (HEAD -> dev, origin/dev)
Author: Anton Kochkov <anton.kochkov@gmail.com>
Date:   Sun Mar 3 14:29:35 2024 +0800

    Update bundled Rizin to the 0.8.0 series
```
* Obtained from:
  - [x] Built from source
  - [ ] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format:
N/A
**Describe the bug**
```
[100%] Linking CXX executable ../cutter
/home/jake/git/cutter/src/widgets/ThreadsWidget.cpp:12:6: error: type ‘ColumnIndex’ violates the C++ One Definition Rule [-Werror=odr]
   12 | enum ColumnIndex {
      |      ^
/home/jake/git/cutter/src/widgets/ProcessesWidget.cpp:12:6: note: an enum with different value name is defined in another translation unit
   12 | enum ColumnIndex {
      |      ^
/home/jake/git/cutter/src/widgets/ThreadsWidget.cpp:14:5: note: name ‘COLUMN_STATUS’ differs from name ‘COLUMN_UID’ defined in another translation unit
   14 |     COLUMN_STATUS,
      |     ^
/home/jake/git/cutter/src/widgets/ProcessesWidget.cpp:14:5: note: mismatching definition
   14 |     COLUMN_UID,
      |     ^
lto1: some warnings being treated as errors
lto-wrapper: fatal error: /usr/bin/c++ returned 1 exit status
compilation terminated.
/usr/lib/gcc/x86_64-pc-linux-gnu/13/../../../../x86_64-pc-linux-gnu/bin/ld: error: lto-wrapper failed
collect2: error: ld returned 1 exit status
make[2]: *** [src/CMakeFiles/Cutter.dir/build.make:2909: cutter] Error 1
make[1]: *** [CMakeFiles/Makefile2:132: src/CMakeFiles/Cutter.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```
<!-- A clear and concise description of what the bug is. -->

**To Reproduce**

Steps to reproduce the behavior:
1. `mkdir build && cd build && CXXFLAGS="-flto -Werror=odr" cmake .. -DCUTTER_USE_BUNDLED_RIZIN=ON && make -j`

**Expected behavior**

<!-- A clear and concise description of what you expected to happen. -->
Build success

**Screenshots**

<!-- If applicable, add screenshots to help explain your problem. -->


**Additional context**

<!-- Add any other context about the problem here. -->

--- a/src/widgets/ProcessesWidget.cpp
+++ b/src/widgets/ProcessesWidget.cpp
@@ -9,13 +9,6 @@
 
 #define DEBUGGED_PID (-1)
 
-enum ColumnIndex {
-    COLUMN_PID = 0,
-    COLUMN_UID,
-    COLUMN_STATUS,
-    COLUMN_PATH,
-};
-
 ProcessesWidget::ProcessesWidget(MainWindow *main)
     : CutterDockWidget(main), ui(new Ui::ProcessesWidget)
 {
@@ -23,10 +16,14 @@ ProcessesWidget::ProcessesWidget(MainWindow *main)
 
     // Setup processes model
     modelProcesses = new QStandardItemModel(1, 4, this);
-    modelProcesses->setHorizontalHeaderItem(COLUMN_PID, new QStandardItem(tr("PID")));
-    modelProcesses->setHorizontalHeaderItem(COLUMN_UID, new QStandardItem(tr("UID")));
-    modelProcesses->setHorizontalHeaderItem(COLUMN_STATUS, new QStandardItem(tr("Status")));
-    modelProcesses->setHorizontalHeaderItem(COLUMN_PATH, new QStandardItem(tr("Path")));
+    modelProcesses->setHorizontalHeaderItem(ProcessesWidget::COLUMN_PID,
+                                            new QStandardItem(tr("PID")));
+    modelProcesses->setHorizontalHeaderItem(ProcessesWidget::COLUMN_UID,
+                                            new QStandardItem(tr("UID")));
+    modelProcesses->setHorizontalHeaderItem(ProcessesWidget::COLUMN_STATUS,
+                                            new QStandardItem(tr("Status")));
+    modelProcesses->setHorizontalHeaderItem(ProcessesWidget::COLUMN_PATH,
+                                            new QStandardItem(tr("Path")));
     ui->viewProcesses->horizontalHeader()->setDefaultAlignment(Qt::AlignLeft | Qt::AlignVCenter);
     ui->viewProcesses->verticalHeader()->setVisible(false);
     ui->viewProcesses->setFont(Config()->getFont());
@@ -129,10 +126,10 @@ void ProcessesWidget::setProcessesGrid()
         rowStatus->setFont(font);
         rowPath->setFont(font);
 
-        modelProcesses->setItem(i, COLUMN_PID, rowPid);
-        modelProcesses->setItem(i, COLUMN_UID, rowUid);
-        modelProcesses->setItem(i, COLUMN_STATUS, rowStatus);
-        modelProcesses->setItem(i, COLUMN_PATH, rowPath);
+        modelProcesses->setItem(i, ProcessesWidget::COLUMN_PID, rowPid);
+        modelProcesses->setItem(i, ProcessesWidget::COLUMN_UID, rowUid);
+        modelProcesses->setItem(i, ProcessesWidget::COLUMN_STATUS, rowStatus);
+        modelProcesses->setItem(i, ProcessesWidget::COLUMN_PATH, rowPath);
         i++;
     }
 
@@ -155,7 +152,7 @@ void ProcessesWidget::onActivated(const QModelIndex &index)
     if (!index.isValid())
         return;
 
-    int pid = modelFilter->data(index.sibling(index.row(), COLUMN_PID)).toInt();
+    int pid = modelFilter->data(index.sibling(index.row(), ProcessesWidget::COLUMN_PID)).toInt();
     // Verify that the selected pid is still in the processes list since dp= will
     // attach to any given id. If it isn't found simply update the UI.
     for (const auto &value : Core()->getAllProcesses()) {
@@ -185,7 +182,7 @@ ProcessesFilterModel::ProcessesFilterModel(QObject *parent) : QSortFilterProxyMo
 bool ProcessesFilterModel::filterAcceptsRow(int row, const QModelIndex &parent) const
 {
     // All columns are checked for a match
-    for (int i = COLUMN_PID; i <= COLUMN_PATH; ++i) {
+    for (int i = ProcessesWidget::COLUMN_PID; i <= ProcessesWidget::COLUMN_PATH; ++i) {
         QModelIndex index = sourceModel()->index(row, i, parent);
         if (qhelpers::filterStringContains(sourceModel()->data(index).toString(), this)) {
             return true;

--- a/src/widgets/ProcessesWidget.h
+++ b/src/widgets/ProcessesWidget.h
@@ -31,6 +31,13 @@ class ProcessesWidget : public CutterDockWidget
     Q_OBJECT
 
 public:
+    enum ColumnIndex {
+        COLUMN_PID = 0,
+        COLUMN_UID,
+        COLUMN_STATUS,
+        COLUMN_PATH,
+    };
+
     explicit ProcessesWidget(MainWindow *main);
     ~ProcessesWidget();
 

--- a/src/widgets/ThreadsWidget.cpp
+++ b/src/widgets/ThreadsWidget.cpp
@@ -9,21 +9,17 @@
 
 #define DEBUGGED_PID (-1)
 
-enum ColumnIndex {
-    COLUMN_PID = 0,
-    COLUMN_STATUS,
-    COLUMN_PATH,
-};
-
 ThreadsWidget::ThreadsWidget(MainWindow *main) : CutterDockWidget(main), ui(new Ui::ThreadsWidget)
 {
     ui->setupUi(this);
 
     // Setup threads model
     modelThreads = new QStandardItemModel(1, 3, this);
-    modelThreads->setHorizontalHeaderItem(COLUMN_PID, new QStandardItem(tr("PID")));
-    modelThreads->setHorizontalHeaderItem(COLUMN_STATUS, new QStandardItem(tr("Status")));
-    modelThreads->setHorizontalHeaderItem(COLUMN_PATH, new QStandardItem(tr("Path")));
+    modelThreads->setHorizontalHeaderItem(ThreadsWidget::COLUMN_PID, new QStandardItem(tr("PID")));
+    modelThreads->setHorizontalHeaderItem(ThreadsWidget::COLUMN_STATUS,
+                                          new QStandardItem(tr("Status")));
+    modelThreads->setHorizontalHeaderItem(ThreadsWidget::COLUMN_PATH,
+                                          new QStandardItem(tr("Path")));
     ui->viewThreads->horizontalHeader()->setDefaultAlignment(Qt::AlignLeft | Qt::AlignVCenter);
     ui->viewThreads->verticalHeader()->setVisible(false);
     ui->viewThreads->setFont(Config()->getFont());
@@ -120,9 +116,9 @@ void ThreadsWidget::setThreadsGrid()
         rowStatus->setFont(font);
         QStandardItem *rowPath = new QStandardItem(path);
         rowPath->setFont(font);
-        modelThreads->setItem(i, COLUMN_PID, rowPid);
-        modelThreads->setItem(i, COLUMN_STATUS, rowStatus);
-        modelThreads->setItem(i, COLUMN_PATH, rowPath);
+        modelThreads->setItem(i, ThreadsWidget::COLUMN_PID, rowPid);
+        modelThreads->setItem(i, ThreadsWidget::COLUMN_STATUS, rowStatus);
+        modelThreads->setItem(i, ThreadsWidget::COLUMN_PATH, rowPath);
         i++;
     }
 
@@ -146,7 +142,7 @@ void ThreadsWidget::onActivated(const QModelIndex &index)
     if (!index.isValid())
         return;
 
-    int tid = modelFilter->data(index.sibling(index.row(), COLUMN_PID)).toInt();
+    int tid = modelFilter->data(index.sibling(index.row(), ThreadsWidget::COLUMN_PID)).toInt();
 
     // Verify that the selected tid is still in the threads list since dpt= will
     // attach to any given id. If it isn't found simply update the UI.
@@ -169,7 +165,7 @@ ThreadsFilterModel::ThreadsFilterModel(QObject *parent) : QSortFilterProxyModel(
 bool ThreadsFilterModel::filterAcceptsRow(int row, const QModelIndex &parent) const
 {
     // All columns are checked for a match
-    for (int i = COLUMN_PID; i <= COLUMN_PATH; ++i) {
+    for (int i = ThreadsWidget::COLUMN_PID; i <= ThreadsWidget::COLUMN_PATH; ++i) {
         QModelIndex index = sourceModel()->index(row, i, parent);
         if (qhelpers::filterStringContains(sourceModel()->data(index).toString(), this)) {
             return true;

--- a/src/widgets/ThreadsWidget.h
+++ b/src/widgets/ThreadsWidget.h
@@ -31,6 +31,12 @@ class ThreadsWidget : public CutterDockWidget
     Q_OBJECT
 
 public:
+    enum ColumnIndex {
+        COLUMN_PID = 0,
+        COLUMN_STATUS,
+        COLUMN_PATH,
+    };
+
     explicit ThreadsWidget(MainWindow *main);
     ~ThreadsWidget();
 

